packages/theme/CHANGELOG.md

```diff
@@ -1,3 +1,7 @@
+# [0.2.0](https://github.com/stackblitz/tutorialkit/compare/0.1.6...0.2.0) "@tutorialkit/theme" (2024-08-28)
+
+
+
```

Comment:
`conventional-changelog` doesn't work with `feat!: ...`. I'll need to update these changelogs manually I think. 

https://github.com/conventional-changelog/conventional-changelog/issues/648

Reply:
Applied https://github.com/conventional-changelog/conventional-changelog/issues/648#issuecomment-704867077 and commits are now detected.